### DISCUSSION TITLE
Increase the size limit of documents

### DISCUSSION THREAD

**Discussion by yandroid** [minio/minio#13250] [Q&A]
Hello,

It is possible to increase the size limit of documents to over than 1MB without build the project?

If yes please explain and if not please explain how to build the minio again into docker image.
Thanks in advance!

**Comment by klauspost** [comment]
No, not as far as I know.

**Reply by harshavardhana** [reply]
> So if I made some changes on global.go how to compile it and create docker image?
> 
> Can you please guide me the steps?
> 
> Thanks a lot!

just do `make docker`

**Reply by klauspost** [reply]
I am not sure you should do this, if you cannot make this work.

**Reply by klauspost** [reply]
We don't provide personal support. If you require that look into our [subscription offer](https://min.io/pricing).

**Reply by harshavardhana** [reply]
Make sure to read this https://github.com/minio/minio/blob/master/CONTRIBUTING.md @yandroid
